RecoTauTag/RecoTau/plugins/RecoTauPiZeroProducer.cc

```diff
@@ -223,8 +223,7 @@ void RecoTauPiZeroProducer::produce(edm::Event& evt, const edm::EventSetup& es)
     if (piZeroMass_ >= 0) {
       std::for_each(
           cleanPiZeros.begin(), cleanPiZeros.end(),
-          std::bind2nd(
-              std::mem_fun_ref(&reco::RecoTauPiZero::setMass), piZeroMass_));
+              [&](auto c){c.setMass(this->piZeroMass_);});
```

Comment:
shouldn't this be ```auto& c``` ?

Reply:
I will replace the whole for_each with a ranged for.